src/components/ClaimMasterPanel.js

```diff
@@ -504,6 +505,19 @@ class ClaimMasterPanel extends FormPanel {
             )}
           </Fragment>
         )}
+        {(this.showPreAuthorization &&
+        <FormControlLabel
+              control={
+                <Checkbox
+                  id="Claim.preAuthorization"
+                  color="primary"
+                  checked={edited?.preAuthorization}
+                  onChange={(e) => this.updateAttribute("preAuthorization", e.target.checked)}
+                />
+              }
+              label={formatMessage(intl, "claim", "pre-authorization")}
```

Comment:
`preAuthorization` instead of `pre-authorization`

Reply:
Done